scalableminds/webknossos
# Fix display of used storage in power plan

### Steps to test:
- Having uploaded some datasets, set `datastore.reportUsedStorage.enabled=true` in `application.conf`
- If needed, edit organization_usedstorage.usedstoragebytes in pgAdmin
- Make sure you're subscribed to any but the Basic Plan (e.g. Custom)
- Go to Administration>Organization page and see whether the used storage is displayed adequately
- Up to 0.1TB of used storage, it should be displayed in GB, otherwise in TB

### Issues:
- fixes #6987 

------
(Please delete unneeded items, merge only when none are left open)
~~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable~~
~~- [ ] Updated [documentation](../blob/master/docs) if applicable~~
~~- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change~~
~~- [ ] Needs datastore update after deployment~~
